src/components/body/body.js

```diff
@@ -171,8 +171,8 @@ var Body = {
 
     var data = this.data;
 
-    if (prevData.type != undefined && data.type !== prevData.type) {
-      console.warn('CANNON.Body type cannot be changed after instantiation');
+    if (prevData.type != undefined && data.type != prevData.type) {
+      this.body.type = data.type === 'dynamic' ? CANNON.Body.DYNAMIC : CANNON.Body.STATIC;
     }
 
     this.body.mass = data.mass || 0;
```

Comment:
Is it valid to have a static body with a non 0 mass? If not it should maybe output a warning here and correct it.

Reply:
It's valid insofar as cannon doesn't care